roles/ceph-common/tasks/main.yml

```diff
@@ -93,10 +88,24 @@
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
 
-- include: facts.yml
-- include: ./checks/check_socket.yml
+- name: get ceph version
```

Comment:
I might have missed something but at the task you highlighted here, you should have `ceph` command available, the task which installs ceph packages are executed earlier in that same role but I'll investigate this.